src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,52 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int capacity = 12;
+    private static final int indexOutOfArray = -1;
+    private int[] numbers = new int[capacity];
 
-    public int total = -1;
+    private int total = getIndexOutOfArray();
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == getIndexOutOfArray();
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == capacity - 1;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return getIndexOutOfArray();
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)